File: languages/go/entity/identityV2/issuer_key_args.go
package identityV2

import "github.com/bloock/bloock-sdk-go/v2/entity/key"

// IssuerKeyArgs represents arguments for configuring an issuer key.
type IssuerKeyArgs struct {
	LocalKey   *key.LocalKey
	ManagedKey *key.ManagedKey
}

File: languages/java/src/main/java/com/bloock/sdk/entity/identity_v2/IssuerKeyArgs.java
package com.bloock.sdk.entity.identity_v2;

import com.bloock.sdk.entity.key.LocalKey;
import com.bloock.sdk.entity.key.ManagedKey;

public class IssuerKeyArgs {

  LocalKey localKey;
  ManagedKey managedKey;

  public IssuerKeyArgs(LocalKey localKey) {
    this.localKey = localKey;
  }

  public IssuerKeyArgs(ManagedKey managedKey) {
    this.managedKey = managedKey;
  }
}
